assets/meshes/teleport/teleport_title_material.tres

```diff
@@ -18,6 +18,7 @@ description = "These nodes add the image and scrolling UVs to sample the title t
 [sub_resource type="VisualShaderNodeTexture2DParameter" id="5"]
 parameter_name = "Title"
 texture_type = 1
+texture_repeat = 1
```

Comment:
Was this a left over from the previous try?

Reply:
No, I threw that branch and pull request away.

It looks like the Visual Editor put this attribute on the node properties for the node-view, and then also updated the embedded copy of the generated shader code.

Reply:
Well if its the visual editor that adds that in then it's fine I guess :)